cmd/utils/selector.go

```diff
@@ -42,6 +42,10 @@ const (
 // ErrInvalidOption is raised when the selector has an invalid option
```

Comment:
NIT: I think we should move this file into another package, but don't know where it should fit. Since we are storing every io under `pkg/log` I think it could potentially fit there

Reply:
i finally moved to `prompt` pkg as i felt this was not really log... but input from the user. Also i moved some other question related functions (Yes/No and question)